### ISSUE TITLE
Show number of slotted players per <company>

### ISSUE THREAD

**Issue by McDiod** [gruppe-adler/nodebb-plugin-arma3-slotting#44]
Useful for:

* comparing number of registered players to number of slotted players
* balancing teams in TvT

**Comment by Fusselwurm** [comment]
good idea. we could generally show the number of slotted players in every sub unit (excepting fireteams – i think it's awkward to add something there, UI wise)

**Comment by nomisum** [comment]
if i get an endpoint i would implement the frontend for this

**Comment by Fusselwurm** [comment]
I *think* I'll just add `groups: ['Adler', 'Gastspieler', …]` to the user models

**Comment by Fusselwurm** [comment]
@nomisum every unit has a `slottedPlayerCount` attribute now, which should be usable in the templates. (Currently, it's only used for the whole match)
